glass/lensing.py

```diff
@@ -473,15 +472,23 @@ def add_plane(
         w2, self.w3 = self.w3, wlens
 
         # extrapolation law
-        x2, self.x3 = self.x3, self.cosmo.xm(self.z3)
+        x2, self.x3 = (
+            self.x3,
+            self.cosmo.hubble_distance
+            * self.cosmo.transverse_comoving_distance(self.z3),
```

Comment:
There's no dimensionless transverse comoving distance in `cosmology.api` so these are necessarily longer